composer.json

```diff
@@ -3,14 +3,14 @@
     "license": "MIT",
     "description": "Add a CSV response type to Laravel",
     "require": {
-        "php": ">=5.6",
-        "illuminate/routing": "^5.1|^6.0|^7.0|^8.0|^9.0",
-        "illuminate/support": "^5.1|^6.0|^7.0|^8.0|^9.0"
+        "php": "^8.0.2",
+        "illuminate/routing": "^9.0|^11.0",
```

Comment:
`|` should be `||` according to composer docs (though `|` still works for backwards compatibility)